.github/workflows/ml-cicd.yml

```
@@ -266,22 +268,22 @@ jobs:
           path: src/ml/models/trained/
 
       - name: Download DuckDB database
         uses: actions/download-artifact@v4
         with:
-          name: duckdb-dwh-${{ github.ref_name }}
-          path: src/ml/data/dwh/
+          name: duckdb-dwh-${{ github.sha }}
+          path: src/ml/data/dwh/data/
 
       - name: Create Release
         uses: softprops/action-gh-release@v1
```

actionlint (1.7.7)

the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)